packages/experiments/src/components/Pagination/examples/Pagination.Customization.Example.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Pagination } from '../Pagination';
+
+export interface IPaginationBasicExampleState {
+  selectedPageIndex: number;
+}
+export class PaginationCustomizaionExample extends React.Component<{}, IPaginationBasicExampleState> {
```

Comment:
Nit: `PaginationCustomizaionExample ` => `PaginationCustomizationExample`

Reply:
addressed

---
In reply to: [222074391](https://github.com/OfficeDev/office-ui-fabric-react/pull/6532#discussion_r222074391) [](ancestors = 222074391)